node/cmd/guardiand/node.go

```diff
@@ -162,6 +163,10 @@ var (
 	optimismRPC      *string
 	optimismContract *string
 
+	// hederaWS       *string
```

Comment:
Delete commented out code? I see that in the watcher as well.

Reply:
Done